recipes/lfpy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = 'LFPy' %}
+{% set version = "2.0.2" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/l/lfpy/{{name}}-{{ version }}.tar.gz
+  sha256: ab5103f6fd5fc9fb4005e65033a98d9d67f9ace4bc4b9be6c3462eab65539698
+
+build:
+  number: {{ build }}
+  script: "{{PYTHON}} -m pip install . --no-deps -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - cython >=0.20
+    - numpy >=1.8
+    - neuron
+  run:
+    - python
+    - setuptools
+    - cython >=0.20
+    - numpy >=1.8
+    - scipy >=0.14
+    - h5py >=2.5
+    - mpi4py >=1.2
+    - neuron
+
+test:
+  requires:
+    # test with mpich instead of openmpi
+    # mpi4py with openmpi doesn't import on CI
+    # without extra environment variables set
+    - mpich
```

Comment:
Should ``mpich`` add as a ``run`` ``requirement`` ?

Reply:
No, it's not a direct run requirement. mpi4py is a direct run requirement, which works with *either* mpich or openmpi. Since lfpy is not linked to mpi (at least not currently), the implementation is not sensitive. This selection is only for the test because both work on a normal machine, but openmpi requires extra environment variables like allowing oversubscribe, etc. when run on CI.